content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vm-dashboard-profile.md

```diff
@@ -1,107 +0,0 @@
----
```

Comment:
Why is this removed?

Reply:
Discussed on a call with @cloudmaniac - VMO pack used to create VMO profile now replaces vm-dashboard profile.